internal/metrics/metrics.go

```diff
@@ -42,9 +43,19 @@ var metricTemplateUsage = prometheus.NewGaugeVec(
 	[]string{metricLabelTemplateKind, metricLabelTemplateName, metricLabelParentKind, metricLabelParentNamespace, metricLabelParentName},
 )
 
+var metricTemplateInvalidity = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: kcm.CoreKCMName,
```

Comment:
Just out of curiosity, is this correct? The value of `kcm.CoreKCMName` is set to `kcm`. Shouldn’t it match the namespace of the k0rdent installation or it's another parameter?

Reply:
It should not. It is the prom ns, read it as a "prefix" for each metric: `kcm_<metric_name>`